README.md

```diff
@@ -524,7 +524,7 @@ The following command will restart all pods in the `web` and `jobs` deployments:
 Add the annotation `shipit.shopify.io/restart` to all the deployments you want to target, like this:
 
 ```yaml
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```

Comment:
If the users are under `v1.16`, apiVersion should still be `apps/v1beta1`, or it's compatible? I'm a little confused

Reply:
This is one of the core changes for `k8s` 1.16, for `deployments, statefulsets, daemonsets and replicasets` require `apps/v1` and will fail when users try to use `apps/v1beta1` with `1.16`.

That is why I had to change everything from `apps/v1beta1` to `apps/v1`, is it clear now?

Reply:
From then readme point of view, if the users are in k8s `v1.15`,  will `apps/v1` work for them?

Reply:
Yes, `apps/v1` works (for some since `k8s` 1.9, this is why our tests pass in buildkite for the 1.11 and above for this branch.

Here is the documentation that mentions it: https://kubernetes.io/blog/2019/09/18/kubernetes-1-16-release-announcement/